Documentation/Syntax/StringFormats/Index.rst

```diff
@@ -0,0 +1,175 @@
+:navigation-title: String formats
+..  include:: /Includes.rst.txt
+..  _data-types:
+..  _data-types-reference:
+..  _string-formats:
+
+=========================
+TypoScript string formats
+=========================
+
+The values assigned to properties in TypoScript are internally handled as
+strings.
+
+Some properties expect a certain format of string. These formats are described
+in this chapter.
+
+Enforcing these string formats is up to the implementation of the property using
+them. They are therefore no real constructs of the TypoScript syntax.
+
+..  contents::
+    :local:
+
+..  index:: Simple data types; boolean
+..  _data-type-boolean:
+
+boolean
+=======
+
+..  confval:: boolean
+    :name: data-type-boolean
+
+    Possible values for boolean variables are `1` and `0`
+    meaning TRUE and FALSE.
+
+    Everything else is `evaluated to one of these values by PHP`__:
+    Non-empty strings (except `0` [zero]) are treated as TRUE,
+    empty strings are evaluated to FALSE.
+
+    __ https://www.php.net/manual/en/language.types.boolean.php
+
+    ..  rubric:: Examples
+
+    .. code-block:: typoscript
+       :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+
+       dummy.enable = 0   # false, preferred notation
+       dummy.enable = 1   # true,  preferred notation
+       dummy.enable =     # false, because the value is empty
+
+..  index:: Simple data types; date-conf
+..  _data-type-date-conf:
+
+date-conf
+=========
+
+..  confval:: date-conf
+    :name: data-type-date-conf
+
+    Used to format a date, see PHP function :php:`date()`. See the
+    documentation of `allowed date time formats in
+    PHP <https://www.php.net/manual/en/datetime.format.php>`__.
+
+    ..  rubric:: Example
+
+    ..  code-block:: typoscript
+        :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+
+        page.10.date-conf = d-m-y
+
+
+..  index:: Simple data types; function name
+..  _data-type-function-name:
+
+function name
+=============
+
+..  confval:: function name
+    :name: data-type-function-name
+
+    Indicates a function or method in a class to call. See more information at
+    the :ref:`USER cObject <cobj-user>`.
+
+    ..  rubric:: Examples
+
+    Method name in a namespaced class:
+
+    ..  code-block:: typoscript
+
+        lib.someThing = USER_INT
+        lib.someThing.userFunc = MyVendor\MyExtension\SomeNamespace\SomeClass->someFunction
+
+    ..  literalinclude:: _FunctionName.php
+        :language: php
+        :caption: EXT:my_extension/Classes/SomeNamespace/SomeClass.php
+
+..  index:: Simple data types; integer
+..  _data-type-positive-integer:
+..  _data-type-integer:
+
+integer
+=======
+
+..  confval:: integer
+    :name: data-type-integer
+
+    ..  rubric:: Examples
```

Comment:
I don't know "rubric":

https://www.google.com/search?q=typo3+docs+rubric&num=10&client=firefox-b-d&sca_esv=b7414eef6243bef6&biw=2560&bih=1330&tbs=li%3A1&ei=C8RNZ4SFBpuK9u8Ph5_B-As&ved=0ahUKEwjEtonlpYmKAxUbhf0HHYdPEL8Q4dUDCA8&uact=5&oq=typo3+docs+rubric&gs_lp=Egxnd3Mtd2l6LXNlcnAiEXR5cG8zIGRvY3MgcnVicmljMgUQIRigATIFECEYoAFIhyJQpAxY0B9wAXgAkAEAmAG2AaAB_QuqAQQwLjExuAEDyAEA-AEBmAIMoAKvDMICChAAGIAEGLADGA3CAgkQABiwAxgNGB7CAgsQABiwAxgIGA0YHsICBRAAGIAEwgIGEAAYFhgewgIEECEYFZgDAIgGAZAGBZIHBDEuMTGgB9wg&sclient=gws-wiz-serp

Reply:
It is a general rst feature: https://rest-sphinx-memo.readthedocs.io/en/latest/ReST.html#rubric

Reply:
Narf ;-) OK, in that case I can search for ages in our docs.